mtags/src/main/scala-2/scala/meta/internal/pc/MetalsGlobal.scala

```diff
@@ -934,9 +934,11 @@ class MetalsGlobal(
         case t => t
       }
     val pre = stabilizedType(qual(tree))
-    val memberType = pre.memberType(symbol)
-    if (memberType.isErroneous) symbol.info
-    else memberType
+    if (pre != null) {
```

Comment:
Could you add the test case you mentioned? It's not an issue if it doesn't handle anything properly yet, we can create an issue for that.